Packs/PaloAltoNetworksCortex_v2/Scripts/GetLicenseID/GetLicenseID_description.md

```diff
@@ -0,0 +1 @@
+This script returns the Demisto license ID
```

Comment:
this file should exist only for integrations, the script description should be in the yml file